templates/pages/index.html

```diff
@@ -5,9 +5,13 @@
 {% block content %}
 	<!-- Main jumbotron for a primary message -->
 	<div class="jumbotron">
-		<h1>TrackCat</h1>
+        <div class="h1"> 
```

Comment:
The div here is created in order to put the h1 inside a bordered block, like here: http://designschool.canva.com/blog/use-shapes-creatively-design/. I was thinkinking about creating a similar design (image for background, bolded title in the block in the middle of the picuture). Any other suggestions for doing that? 
![screen shot 2015-01-10 at 10 03 45](https://cloud.githubusercontent.com/assets/10223254/5690987/49327cba-98b0-11e4-95ce-8e6fadd9b730.png)